website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -228,6 +228,15 @@ See the complete migrated `memcached_controller.go` code [here][memcached_contro
 
 **Note:** The version of [controller-runtime][controller-runtime] used in the projects scaffolded by SDK `0.19.x+` was `v0.6.0`. Please check [sigs.k8s.io/controller-runtime release docs from 0.7.0+ version][controller-runtime] for breaking changes.
 
+##### Updating your ServiceAccount in Go operator projects
```

Comment:
@estroz I just realized that `in Go operator projects` is not required here. See that it is the Golang migration guide so it is all about Go-based operators. Also, should we not add any info in the https://master.sdk.operatorframework.io/docs/building-operators/golang/operator-scope/ docs as well?
Also, I think we need to update ansible and helm before 1.5 regards this one too for doing no start o b very confuse.